jp.yml

```diff
@@ -0,0 +1,279 @@
+      expandLines_other: "隠された{{count}}行を展開"
+  home:
+    browse: "Modをブラウズ"
+    installedMods:
+      title: "インストールされたMod"
+      noMods: "modはインストールされていません"
+    featuredMods:
+      title: "注目のMod"
+      noMods: "まだインストールされていない注目のmodはありません"
+      explore: "他のModを探す"
+  explore:
+    search:
+      placeholder: "mod検索中..."
+      popularAndTopRated: "人気で評価が高い"
+      popular: "人気"
+      topRated: "評価が高い"
+      newest: "最新"
+      lastUpdated: "最終更新日"
+      alphabeticalOrder: "アルファベット順"
+  settings:
+    language:
+      title: "言語"
+      description: "Windhawkのお好みの表示言語を選択してください。"
+      contribute: "<0>新しい翻訳に貢献する</0>."
+      credits: "<0>lmncaj</0>によって日本語翻訳されました。"
+      creditsLink: "mailto:john.smith@example.com"
+    updates:
+      title: "アップデートを確認する"
+      description: "定期的に新バージョンのWindhawkとインストールされているmodを確認する。"
+    devMode:
+      title: "開発者モード"
+      description: "modの作成・修正など、開発者向けのアクションを表示します。"
+    advancedSettings: "詳細設定"
+    hideTrayIcon:
+      title: "トレイアイコンを隠す"
+      description: "Windhawkを終了するには、このオプションを無効にする必要があります。"
+    requireElevation:
+      title: "Windhawkの実行にはUACの昇格が必要です"
+      description: "Windhawkは管理者権限を必要としますが、シングルユーザーのコンピュータでは、毎回UACプロンプトが表示されるのは煩わしいので、Windhawkはこれを回避します。このオプションを有効にすると、Windhawkの実行にUACの昇格が必要になります。"
+    modInitDialogDelay:
+      title: "Mod初期化ダイアログの遅延"
+      description: "modの初期化時にプログレスダイアログを表示するまでの待ち時間(ミリ秒)を指定します。"
+    moreAdvancedSettings:
+      title: "さらに詳細設定を表示"
+      restartNotice: "設定を適用するためにWindhawkが再起動されます。"
+      saveButton: "保存してWindhawkを再起動する"
+      cancelButton: "キャンセル"
+    loggingVerbosity:
+      appLoggingTitle: "Windhawkログの冗長性"
+      engineLoggingTitle: "Windhawkエンジンのログの冗長性"
+      description: "ログはDebugViewなどのツールで見ることができます"
+      none: "なし"
+      error: "エラー"
+      verbose: "冗長"
+    processList:
+      titleExclusion: "プロセス除外リスト"
+      descriptionExclusion: "Windhawkがインジェクトしないプロセス名/パスのリストです。Windhawkが特定のプログラムと互換性がない場合に便利です。このリストにプロセスを追加すると、Windhawkがそれをカスタマイズするのを防ぐだけでなく、このプロセスが作成する子プロセスをWindhawkが傍受するのを防ぐことに注意してください。このため、このような場合、Windhawkはmodの読み込みに若干の遅れを生じます。"
+      titleInclusion: "プロセス包含リスト"
+      descriptionInclusion: "除外リストにある場合でも、Windhawkがインジェクトするプロセス名/パスのリストです。"
+      inclusionWithoutExclusionNotice: 'プロセス包含リストは、プロセス除外リストが空である場合には、何の効果もありません。これらのプロセス以外を除外したい場合は、プロセス除外リストに "*"を設定します。'
+      titleThreadAttachExempt: '"light threads"インジェクションメソッドのプロセスリストです'
+      descriptionThreadAttachExempt: "Windhawkスレッドの初期化の一部をスキップします（TLSの初期化、TLSおよびDllMainコールバックなどをスキップします）。これにより、一部のプログラムとの互換性が修正されるかもしれませんが、他のプログラムとの互換性が壊れるかもしれません。 <0>さらなる情報</0>."
+      processListPlaceholder: "プロセス名/パス、1行に1つずつ、例:"
+    injectIntoCriticalProcesses:
+      title: "重要なシステムプロセスにインジェクトする"
+      description: "デフォルトでは、Windhawkは重要なシステムプロセスにはコードをインジェクトしません。これらのプロセスの1つをクラッシュさせるとBSODにつながる可能性があるため、このオプションは注意して使用してください。"
+  about:
+    title: "Windhawk v{{version}}"
+    beta: "ベータ"
+    subtitle: "Windowsプログラムのカスタマイズマーケットプレイス"
+    credit: "By <0>{{author}}</0>"
```

Comment:
Untranslated.

Reply:
I think I don't need to translate 「Windhawk」.
also, By is common usage in Japanese too, so it'll become strange if it translated.